pynamodb/models.py

```diff
@@ -169,40 +171,104 @@ class MetaModel(AttributeContainerMeta):
     """
     def __init__(cls, name, bases, attrs):
         super(MetaModel, cls).__init__(name, bases, attrs)
+
+        # We don't' allow inheritance from multiple direct parents
+        if (len(bases) > 1):
+            raise InheritanceError("Model can't inherit from multiple direct parents")
+
+        parent = None if len(bases) == 0 else bases[0]
+        # TODO: special parents as constants
+        parent_is_attribute_container = parent.__name__ == 'AttributeContainer'
+        parent_is_model = parent.__name__ == 'Model'
+        parent_meta_class = None
+        is_parent_abstract = None
+        if parent  and not parent_is_attribute_container:
+            parent_meta_class = getattr(parent, META_CLASS_NAME, None)
+        if parent_meta_class and hasattr(parent_meta_class, ABSTRACT):
+            is_parent_abstract = getattr(parent_meta_class, ABSTRACT)
+
+        # Temporary check until implementing other inheritance types
```

Comment:
It seems like the thing to do here is to iterate through bases and check if any of them are `Model` types, and if they are, throw an exception if any of them don't have the `abstract=true` field in their `Meta` class